extensions/anonaddy/package.json

```diff
@@ -14,46 +14,53 @@
   ],
   "license": "MIT",
   "commands": [
+    {
+      "name": "create-alias",
+      "title": "Create Alias",
+      "description": "Create a new Addy email alias with customizable options.",
+      "mode": "view"
+    },
     {
       "name": "generate-alias",
       "title": "Generate Alias",
-      "subtitle": "Addy",
-      "description": "Generate a random Addy alias",
+      "description": "Quickly generate an Addy email alias using your default settings.",
       "mode": "no-view"
     },
     {
-      "name": "list-aliases",
```

Comment:
Can we keep the old `name`? It is the unique ID of the command where ranking, aliases, and hotkeys are saved. You can, of course, change the title 🙂

Reply:
Fixed!